trax/supervised/configs/bert.gin

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 The Trax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import t5.data.preprocessors
+import trax.layers
+import trax.models
+import trax.data
+import trax.optimizers
+import trax.supervised.lr_schedules
+import trax.supervised.trainer_lib
+import trax.layers.metrics
+
+# See https://www.tensorflow.org/datasets/catalog/glue -- valid dataset_name
```

Comment:
does this comment block make sense for bert.gin? given that dataset_name doesn't exist here ... its OK to replicate across the child gin files I think.